src/layer/__tests__/fixtures.findTiles.ts

```diff
@@ -8,8 +8,8 @@ import {
   AcquisitionMode,
   S1GRDAWSEULayer,
   BBox,
-} from 'src';
-import { CRS_EPSG4326 } from 'src/crs';
+} from '../../index';
+import { CRS_EPSG4326 } from '../../crs';
```

Comment:
Since CRS_EPSG4326 is exported in the src/index.ts, could it be in the import above?

Reply:
Changed

Reply:
Resolving